mlir/test/Integration/GPU/CUDA/sm90/tma_load_64x64_swizzle128b.mlir

```diff
@@ -0,0 +1,247 @@
+// RUN: mlir-opt %s --convert-nvgpu-to-nvvm \
+// RUN:         -convert-linalg-to-loops \
+// RUN:         -canonicalize -cse \
+// RUN:         -gpu-kernel-outlining \
+// RUN:         -canonicalize -cse \
+// RUN:         -convert-vector-to-scf  \
+// RUN:         -canonicalize -cse \
+// RUN:         -lower-affine \
+// RUN:         -canonicalize -cse \
+// RUN:         -convert-scf-to-cf \
+// RUN:         -canonicalize -cse \
+// RUN:         -convert-nvvm-to-llvm \
+// RUN:         -canonicalize -cse \
+// RUN:         -convert-nvgpu-to-nvvm \
+// RUN:         -canonicalize -cse \
+// RUN:         -convert-scf-to-cf  \
+// RUN:         -convert-vector-to-llvm \
+// RUN:         -canonicalize -cse \
+// RUN:         -convert-math-to-llvm \
+// RUN:         -canonicalize -cse \
+// RUN:         -lower-affine \
+// RUN:         -convert-index-to-llvm=index-bitwidth=32 \
+// RUN:         -convert-arith-to-llvm \
+// RUN:         -finalize-memref-to-llvm='use-opaque-pointers=1' \
+// RUN:         -convert-func-to-llvm \
+// RUN:         -canonicalize -cse \
```

Comment:
This pipeline really seems messy: there seems like a clear abuse of "canonicalize, cse", and the overall ordering isn't clear to me (why is `convert-nvvm-to-llvm` before `convert-nvgpu-to-nvvm`? Why is `convert-scf-to-cf` duplicated?)

Also I would think that the `convert-to-llvm` should cover most of of the LLVM conversion.


, can you please clean this up?


 and make this a single `mlir-opt` invocation being a `--pass-pipeline`

Reply:
Thanks for bringing it to my attention. Indeed there were redundancies in the pass pipeline. I clean them up in #67416